hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -17,7 +17,7 @@
 
             **Checks**": Describe the self-check procedure.
             - add bullets as needed
-            - second bullet<br>
+            - second bullet \
```

Comment:
Is the space before "\" needed?

Reply:
Not necessarily in this PR, but I suggest we try to render this as markdown (rather than inventing yet another not-quite-specified markup language). The reggen code has a similar problem that I punted on - maybe we can fix everything at once :-)

Reply:
Please resolve